packages/react-components/README.md

```diff
@@ -11,7 +11,7 @@ on the expected network while taking into account that the user may decide to sw
 ### [`WithWalletConnector`](./src/WithWalletConnector.ts)
 
 Component that bridges [`@concordium/wallet-connectors`](../wallet-connectors) into a React context by
-managing active connector, connection, connected account, network information, errors, etc. in its internal state.
+managing connection state and network information as component state.
```

Comment:
```suggestion
managing connection state and network information.
```
I just think "...as component" state sounds weird, and the sentence works without it.